Makefile

```diff
@@ -20,6 +20,6 @@ test: $(SRC_C) $(SRC_H) tests/*.c
 	./$@
 
 clean:
-	rm -f test hyperfused
```

Comment:
this should actually be test and only -f. `test` is build by doing `make test`. i can see now how it might be a bit confusing